src/content/third_party_migration/vendors/redhat-json.json

```diff
@@ -0,0 +1,30 @@
+{
+    "vendor": "redhat-json",
+    "description": "RedHat JSON",
+    "remotes": [
+        {
+            "url": "https://access.redhat.com/labs/securitydataapi/cve.json?after={after}&per_page={per_page}&page={page}",
+            "request": "api",
+            "type": "json",
+            "parameters": {
+                "after": {
+                    "type": "fixed",
+                    "description": "Date in ISO 8601 format (YYYY-MM-DD or YYYYMMDD)",
+                    "value": "1999-01-01"
+                },
+                "per_page": {
+                    "type": "fixed",
+                    "description": "Vulnerabilities per page",
+                    "value": "1000"
+                },
+                "page": {
+                    "type": "variable-incremental",
+                    "value": [
+                        "1",
+                        "inf"
```

Comment:
- To make the schema  and the logic on the fetchers simpler:
    - Change `value` from array to object.
    - Add the following fields:
        - Start: String that contains solely numbers
        - End: String that contains solely numbers or keyword inf
        - Step (optional): String that contains solely numbers